qcodes/instrument_drivers/stanford_research/SR86x.py

```diff
@@ -1,25 +1,26 @@
 import numpy as np
-import time
 import logging
-from typing import Optional, Sequence, Dict
+from typing import Sequence, Dict, Callable
 
 from qcodes import VisaInstrument
 from qcodes.instrument.channel import InstrumentChannel
 from qcodes.utils.validators import Numbers, Ints, Enum
 from qcodes.instrument.parameter import ArrayParameter
 
+
 log = logging.getLogger(__name__)
 
 
 class SR86xBufferReadout(ArrayParameter):
     """
-    The parameter array that holds read out data. We need this to be compatible with qcodes.Measure
-
-    Args
-    ----
-        name (str)
-        instrument (SR86x): This argument is unused, but needed because the add_parameter method of the Instrument
-                            base class adds this as a kwarg.
+    The parameter array that holds read out data. We need this to be compatible
+    with qcodes.Measure
+
+    Args:
+        name
+        instrument
+            This argument is unused, but needed because the add_parameter
+            method of the Instrument base class adds this as a kwarg.
```

Comment:
I don't understand why the instrument is just not passed to the super method so that the paramter is bound to the instrument. (I realize this is nothing new)

Reply:
indeed. smth to fix later